modules/internal/ChapelDomain.chpl

```diff
@@ -1262,26 +1262,13 @@ module ChapelDomain {
       for i in _value.dimIter(d, ind) do yield i;
     }
 
-   /* Return a tuple of :proc:`intIdxType` describing the size of each
-      dimension.  Note that this routine is in the process of changing
-      to return a tuple of ``int``, similar to the ``.size`` query on
-      ranges, domains, and arrays.  See :proc:`ChapelRange.range.size` for details.
+   /* Return a tuple of `int` describing the size of each
```

Comment:
I think this should be wrapped by two ` on either side, like the removed reference in the third line.

Reply:
Good catch, thanks!  Re-reading this, I'm also rewording as:

```
Return a tuple of ``int`` values representing the size of each dimension, which seems a bit more precise.
```